<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileImport</class>
 <widget class="QWizard" name="FileImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File import wizard</string>
  </property>
  <widget class="QWizardPage" name="page1">
   <property name="title">
    <string>Import files in the database</string>
   </property>
   <property name="subTitle">
    <string>Please select the files you want to import.</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>870</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>15</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="input_src_path">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radio_folder">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Import a folder</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">radio_is_folder</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radio_file">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Import files</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">radio_is_folder</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_browse_src">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="QWidget" name="site_manual" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_year">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Site</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_site">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_plot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="QWidget" name="site_auto" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Define the order in which the information is stocked on the hard drive starting with the topmost folder:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Top folder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="idx_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Year</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Site</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Plot</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Subfolder 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="idx_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Year</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Site</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Plot</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Subfolder 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="idx_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>Year</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Site</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Plot</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Each file requires information about site, year and plot:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QRadioButton" name="radio_site_auto">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Automatically detect information from folder hierarchy. The 3 folders above the audio files will be used to extract the information</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">radio_site_info</string>
       </attribute>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QRadioButton" name="radio_site_manual">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Manually enter the relevant information (does not work with folder import with subfolders as multiple sites could be imported this way)</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">radio_site_info</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>File type:</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_rec_auto">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Auto-detect</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radio_recorders</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_rec_sm">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>SongMeter</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radio_recorders</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_rec_am">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Audiomoth</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radio_recorders</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QCheckBox" name="checkbox_subfolders">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Include subfolders</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page2">
   <property name="title">
    <string>Review selected files</string>
   </property>
   <property name="subTitle">
    <string>Here you can see what files have been selected and what information will be imported into the database</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_status">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QTableView" name="table_files">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ContiguousSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page3">
   <property name="title">
    <string>File management options</string>
   </property>
   <property name="subTitle">
    <string>Please select additional import options</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>851</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Overwrite existing files</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Move and rename files</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Compression options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="1">
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>tar</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">radio_compression</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>ZIP</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">radio_compression</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>7z (Recommended)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">radio_compression</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Use the following compression format:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Compress wav files to FLAC</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Create a compressed archive of the files</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page4"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radio_recorders"/>
  <buttongroup name="radio_is_folder"/>
  <buttongroup name="radio_site_info"/>
  <buttongroup name="radio_compression"/>
 </buttongroups>
</ui>
